final release/format_data.py

```diff
@@ -1,200 +1,266 @@
+    fri_in_asu = sep_i <= 5 * u.arcsec
+    frii_in_asu = sep_ii <= 5 * u.arcsec
 
     # Extacting FR images that are in asu
     asu_fri = fri_images[fri_in_asu]
     asu_frii = frii_images[frii_in_asu]
 
-    leftover_fri = fri_images[np.invert(fri_in_asu)]
-    leftover_frii = frii_images[np.invert(frii_in_asu)]
+    leftover_fri = fri_images[~(fri_in_asu)]
+    leftover_frii = frii_images[~(frii_in_asu)]
+ 
+    # Generating Labels
+    asu_fri_labels = np.zeros(asu_fri.shape[0], dtype=bool)
+    asu_frii_labels = np.ones(asu_frii.shape[0], dtype=bool)
 
-    #Generating Labels
-    asu_fri_lab = np.zeros(asu_fri.shape[0], dtype=bool)
-    asu_frii_lab = np.ones(asu_frii.shape[0], dtype=bool)
-
-    leftover_fri_lab = np.zeros(leftover_fri.shape[0], dtype=bool)
-    leftover_frii_lab = np.ones(leftover_frii.shape[0], dtype=bool)
+    leftover_fri_labels = np.zeros(leftover_fri.shape[0], dtype=bool)
+    leftover_frii_labels = np.ones(leftover_frii.shape[0], dtype=bool)
 
     # Combining FRI's and FRII's and shuffling
     asu_fr = np.vstack((asu_fri, asu_frii))
-    asu_fr_lab = np.concatenate((asu_fri_lab, asu_frii_lab), axis=0) 
+    asu_fr_lab = np.concatenate((asu_fri_labels, asu_frii_labels), axis=0) 
 
     leftover_fr = np.vstack((leftover_fri, leftover_frii))
-    leftover_fr_lab = np.concatenate((leftover_fri_lab, leftover_frii_lab), axis=0)             
-
-    #Shuffling
-    asu_fr_, asu_fr_lab_ = shuffle(asu_fr, asu_fr_lab, random_state = seed)  
-    leftover_fr, leftover_fr_lab = shuffle(leftover_fr, leftover_fr_lab, random_state = seed) 
+    leftover_fr_lab = np.concatenate((leftover_fri_labels, leftover_frii_labels), axis=0)             
+    # Shuffling
+    asu_fr, asu_fr_labels = shuffle(asu_fr, asu_fr_labels, random_state=seed)
+    
+    leftover_fr, leftover_fr_labels = shuffle(leftover_fr, leftover_fr_labels, random_state=seed) 
 
     # Setting training and testing sets
-    if flip_train_test == False:
-        train_x = asu_fr_
-        train_y = asu_fr_lab_
+    if not flip_train_test:
+        train_x = asu_fr
+        train_y = asu_fr_labels
 
         test_x = leftover_fr
-        test_y = leftover_fr_lab
-
+        test_y = leftover_fr_labels
 
-    elif flip_train_test == True:
+    else:
         train_x = leftover_fr
-        train_y = leftover_fr_lab
+        train_y = leftover_fr_labels
 
-        test_x = asu_fr_
-        test_y = asu_fr_lab_
-
-    
-    # Defining rescaling function
-    def rescale_imgs(images,dim=dim):
-        size = images.shape[0]
-        imgs = np.zeros((size, dim, dim))
-
-        for i in range(size):
-            imgs[i, :, :] = resize(images[i, :, :], (dim, dim))
-
-        return np.expand_dims(imgs, axis=3)
-
-    
-    
+        test_x = asu_fr
+        test_y = asu_fr_labels
 
     #Rescaling Images
-    sys.stdout = open(os.devnull, "w") # don't show printed output
+    sys.stdout = open(os.devnull, "w")  # don't show printed output
```

Comment:
Did you not find a better way to suppress the output?

Reply:
no

Reply:
I've the exhausted the extent of my googling ability

Reply:
What's actually outputting?